packages/api-client/src/components/ViewLayout/ViewLayoutSection.vue

```diff
@@ -9,7 +9,7 @@ const id = nanoid()
     class="flex h-full xl:min-w-0 xl:flex-1 flex-col xl:custom-scroll bg-b-1">
     <div
       :id="id"
-      class="min-h-11 xl:min-h-header flex items-center border-b-1/2 px-3.5 py-1.5 md:px-4 md:py-1.5 xl:px-6 xl:pr-5 text-sm font-medium sticky top-0 bg-b-1 xl:rounded-none group">
+      class="min-h-11 xl:min-h-header flex items-center border-b-1/2 px-3.5 py-1.5 md:px-4 md:py-2.5 xl:px-6 xl:pr-5 text-sm font-medium sticky top-0 bg-b- xl:rounded-none">
```

Comment:
Looks like you lost a character
```suggestion
      class="min-h-11 xl:min-h-header flex items-center border-b-1/2 px-3.5 py-1.5 md:px-4 md:py-2.5 xl:px-6 xl:pr-5 text-sm font-medium sticky top-0 bg-b-1 xl:rounded-none">
```

Reply:
omg, good catch ty